pkg/controllers/secret/secret_controller.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2022 pbrookes@redhat.com.
```

Comment:
we should prob remove this header and set it similar to the one in KCP
``` Copyright 2021 The MultiCluster Traffic Controller Authors.``` and leave the apache license part

Reply:
👍